mkosi/installer/pacman.py

```diff
@@ -16,12 +17,12 @@
 from mkosi.util import umask
 from mkosi.versioncomp import GenericVersion
 
+@dataclasses.dataclass
+class PacmanRepository:
+    id: str
+    url: str
```

Comment:
```suggestion
    url: str

```

Reply:
This one is still missing the empty line.

Reply:
I think it's good. I see two empty lines above and below `PacmanRepository`.

Reply:
You're right, somehow it's stuck for me on GitHub, but in the checkout it's fine.